src/Tizen.NUI.Components/Controls/ItemsView/CollectionView.cs

```diff
@@ -0,0 +1,815 @@
+
+                float left = nextFocusedView.Position.X;
+                float right = nextFocusedView.Position.X + nextFocusedView.Size.Width;
+                float top = nextFocusedView.Position.Y;
+                float bottom = nextFocusedView.Position.Y + nextFocusedView.Size.Height;
+
+                float visibleRectangleLeft = -scrollPosition.X;
+                float visibleRectangleRight = -scrollPosition.X + Size.Width;
+                float visibleRectangleTop = -scrollPosition.Y;
+                float visibleRectangleBottom = -scrollPosition.Y + Size.Height;
+
+                if (ScrollingDirection == Direction.Horizontal)
+                {
+                    if ((direction == View.FocusDirection.Left || direction == View.FocusDirection.Up) && left < visibleRectangleLeft)
+                    {
+                        targetPosition = left;
+                    }
+                    else if ((direction == View.FocusDirection.Right || direction == View.FocusDirection.Down) && right > visibleRectangleRight)
+                    {
+                        targetPosition = right - Size.Width;
+                    }
+                }
+                else
+                {
+                    if ((direction == View.FocusDirection.Up || direction == View.FocusDirection.Left) && top < visibleRectangleTop)
+                    {
+                        targetPosition = top;
+                    }
+                    else if ((direction == View.FocusDirection.Down || direction == View.FocusDirection.Right) && bottom > visibleRectangleBottom)
+                    {
+                        targetPosition = bottom - Size.Height;
+                    }
+                }
+
+                focusedView = nextFocusedView;
+                prevFocusedDataIndex = (nextFocusedView as RecycleItem).DataIndex;
+
+                ScrollTo(targetPosition, true);
+            }
+            else
+            {
+                // If nextView is null, it means that we should move focus to outside of Control.
+                // Return FocusableView depending on direction.
+                switch (direction)
+                {
+                    case View.FocusDirection.Left:
+                    {
+                        nextFocusedView = LeftFocusableView;
+                        break;
+                    }
+                    case View.FocusDirection.Right:
+                    {
+                        nextFocusedView = RightFocusableView;
+                        break;
+                    }
+                    case View.FocusDirection.Up:
+                    {
+                        nextFocusedView = UpFocusableView;
+                        break;
+                    }
+                    case View.FocusDirection.Down:
+                    {
+                        nextFocusedView = DownFocusableView;
+                        break;
+                    }
+                }
+
+                if(nextFocusedView)
```

Comment:
BaseHandle의 operator overloading에 의해 지원하는 방법이긴 하지만 Svace 등의 툴에서 감지하지 못해 Svace warning이 발생합니다. 따라서, NUI에서는 명시적으로 null과 비교하는 것을 권장하고 있습니다.
```suggestion
                if (nextFocusedView != null)
```